worlds/noita/Regions.py

```diff
@@ -96,12 +97,12 @@ def create_all_regions_and_connections(multiworld: MultiWorld, player: int) -> N
     "Lava Lake": {"Mines", "Abyss Orb Room"},
     "Abyss Orb Room": {"Lava Lake"},
     "Below Lava Lake": {"Snowy Depths"},
-    "Dark Cave": {"Ancient Laboratory", "Collapsed Mines"},
+    "Dark Cave": {"Collapsed Mines"},
     "Ancient Laboratory": {"Dark Cave"},

```

Comment:
Is it fine if this still links to Dark Cave?

Reply:
Probably not, woops